src/server/controller/cycleData/updateDependencies/updateCalculationDependencies/updateCalculationDependencies.ts

```diff
@@ -21,7 +21,7 @@ export const updateCalculationDependencies = async (
     { assessment, cycle, variableName, tableName, colName, countryIso, isODP, type: 'calculations' },
     client
   )
-  const visitedVariables: Array<VariableCache> = [{ variableName, tableName }]
+  const visitedVariables: Array<VariableCache> = []
```

Comment:
Remove the skip of current row
This caused problem:

https://user-images.githubusercontent.com/5508251/220076753-0a981d2a-f5df-4804-b1b0-060b04621e09.mov

Reply:
# The resolution
```
const visitedVariables: Array<VariableCache> = []
```

Still WIP